api/v1/storage-contract.go

```diff
@@ -3,6 +3,7 @@ package api
 import (
 	"net/http"
 
+	"github.com/data-preservation-programs/spade-tenant/db"
 	"github.com/labstack/echo/v4"
 )
 
```

Comment:
<i class='graphite__hidden'>[Re: lines 10 to 10]<br/></i>

@jcace what do these objects map to in the model?<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/data-preservation-programs/spade-tenant/11?utm_source=unchanged-line-comment">Graphite</a>.</span>